### PR TITLE
Update

### PR DESCRIPTION
**Pull request by danieldegroot2**
_(Submitted with GitHub browser version); Validate manually._

Initial commit to update some community resources.

### CODE REVIEW

File: resources/europe/united_kingdom/aberdeen-discord.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "aberdeen-discord",
+  "type": "discord",
+  "account": "vY7KBZx2M9",
+  "locationSet": {"include": [[-2.09375, 57.14824]]},
+  "languageCodes": ["en"],
+  "strings": {"name": "OSM ABZ Discord"}
+  ]
```
Comment (bhousel) [inline resources/europe/united_kingdom/aberdeen-discord.json:8]: extra ']' here

**Review by bhousel — CHANGES_REQUESTED** [review]
looks mostly good!  let me know when you're finished and I'll merge it 👍

**Comment by danieldegroot2** [comment]
@bhousel you're welcome to validate and merge the PR.

**Comment by danieldegroot2** [comment]
Noteably, I named all new facebook groups "FB Group", but it could be "Facebook group" if needed since overflow is handled fine.

**Review by bhousel — COMMENTED** [review]
I started going through this but - it's just a lot.

@danieldegroot2 are you ok if I just merge as-is and, maybe walk back some of the changes to an earlier version. For some things where a file got deleted and another one got added in its place, it's really hard to follow what's going on.

File: resources/north-america/united_states/us-newyork-mailinglist.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "id": "us-newyork-mailinglist",
+  "type": "mailinglist",
+  "account": "talk-us-newyork",
+  "locationSet": {"include": ["Q1384"]},
```
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/north-america/united_states/us-newyork-mailinglist.json:5]: Ok. I had this on the troubleshooting list already, it should be a polygon for New York State.

File: resources/asia/indonesia/id-facebook.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "id": "id-facebook",
+  "type": "facebook",
+  "account": "openstreetmap.id",
+  "locationSet": {"include": ["id"]},
+  "order": 1,
+  "strings": {
+    "community": "OpenStreetMap Indonesia",
+    "extendedDescription": "Mapping in Indonesia? Have questions, want to connect with the community here? Join us at {url}. All are welcome!",
+    "url": "https://www.facebook.com/openstreetmap.id/"
```
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/asia/indonesia/id-facebook.json:10]: Sure, I'll add it to the troubleshooting list.

File: resources/north-america/united_states/Western-Slope-facebook.json [hunk 4]
```diff
@@ -1,15 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/north-america/united_states/Western-Slope-facebook.json:1]: See the [commit description](https://github.com/danieldegroot2/osm-community-index/commit/1995009c1eb537cbeaec7d4eba372e02b65004d2).

File: resources/europe/hungary/hu-facebook-group.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "id": "hu-facebook",
+  "type": "facebook",
+  "account": "osm.hu",
+  "locationSet": {"include": ["hu"]},
+  "languageCodes": ["hu"],
+  "order": 3,
+  "strings": {
+    "community": "OpenStreetMap Hungary FB Group",
```
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/europe/hungary/hu-facebook-group.json:9]: This is also already on the troubleshooting list; half of the time they come with a regular facebook page, where it's somewhat redundant for people to understand it's facebook. As I said in the first post, it would be fine to write them out fully, though it may clutter the list slightly because of overflow into another line.

File: resources/asia/indonesia/id-facebook-group.json [hunk 6]
```diff
@@ -1,10 +1,11 @@
 {
-  "id": "id-facebook",
+  "id": "id-facebook-group",
   "type": "facebook",
   "account": "osm.id",
   "locationSet": {"include": ["id"]},
   "order": 1,
   "strings": {
+    "name": "OpenStreetMap Indonesia FB Group",
```
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/asia/indonesia/id-facebook-group.json:8]: There's some resources which had this outside of my PR I think, but I'm not sure. I should add it to the troubleshooting list, anyways.

**Comment by danieldegroot2** [comment]
> I started going through this but - it's just a lot.
> 
> @danieldegroot2 are you ok if I just merge as-is and, maybe walk back some of the changes to an earlier version. For some things where a file got deleted and another one got added in its place, it's really hard to follow what's going on.

Yes, you can merge it. If there's anything which makes it not build, let me know.

Usually it's a case of
- Updated resources: Important ones I've contacted the owners for.
- Facebook \ Facebook group: It's a case of one already being there but under the wrong "id".

For inactive resources, see the commit description of the first deletion. Most I contacted the owners for.

PS: Feel free to have a look at the [backlog](https://github.com/osmlab/osm-community-index/issues/532).

File: resources/europe/hungary/hu-facebook-group.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (bhousel) [reply resources/europe/hungary/hu-facebook-group.json:9]: What troubleshooting list do you mean?

[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/europe/hungary/hu-facebook-group.json:9]: The one [in the first post](https://github.com/osmlab/osm-community-index/pull/524#issue-1190321619).

**Comment by bhousel** [comment]
> Moved notes from this comment to the first post.

Please stop this.  It makes it impossible to follow your discussion if you keep editing your comments.

**Comment by danieldegroot2** [comment]
@bhousel Ready.

File: resources/asia/indonesia/id-facebook.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/asia/indonesia/id-facebook.json]: Resolved in https://github.com/danieldegroot2/osm-community-index/commit/b8fe983f32bcb6036094199139b3e33698122610 and other commits.

File: resources/europe/hungary/hu-facebook-group.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/europe/hungary/hu-facebook-group.json]: Resolved in https://github.com/danieldegroot2/osm-community-index/commit/33079cf7be900e102f7cef4d877f5241c3703c76 and other commits.

File: resources/asia/indonesia/id-facebook-group.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (danieldegroot2) [reply resources/asia/indonesia/id-facebook-group.json]: Resolved in https://github.com/danieldegroot2/osm-community-index/commit/cc12f0ac576cf905b857d96665e16fb5dc41cbac and other commits.

**Comment by bhousel** [comment]
Ok Thanks again @danieldegroot2  -  I just cleaned up the remaining few syntax errors and merged this.. Going forward please try to keep your pull requests small, and don't rewrite your comments - this makes it really hard for people to follow what you are doing.

I also really encourage you to set up a local development environment and get in the habit of running `npm run build` sometimes, as this will check the files for syntax errors.
